sdk/data/azcosmos/cosmos_client.go

```diff
@@ -124,6 +125,7 @@ func newPipeline(authPolicy policy.Policy, gem *globalEndpointManager, options *
 			},
 			PerRetry: []policy.Policy{
 				authPolicy,
+				&clientRetryPolicy{gem: gem},
```

Comment:
Question: Should this policy be in the `PerCall` section instead of the `PerRetry`?